<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cowsFilter</class>
 <widget class="QDialog" name="cowsFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Cow filter</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QComboBox" name="BreedCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="LineCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="NameCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="InvCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="NumberCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="MarkCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="ValidityCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="FamilyCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="BPlaceCombo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="FCalvingCombo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="OwnerCombo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="PurposeCombo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="ColourCombo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="BGroupCombo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="BDateCombo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="InDateCombo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="InvNumTxt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2147483647</number>
   </property>
  </widget>
  <widget class="QLabel" name="InvNumLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>InvNumber</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValidityTxt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="FamilyLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Family</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumberTxt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2147483647</number>
   </property>
  </widget>
  <widget class="QLabel" name="NumberLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineTxt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="BreedTxt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="ValidityLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Validity</string>
   </property>
  </widget>
  <widget class="QLabel" name="BreedLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Breed</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FamilyTxt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LineLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </widget>
  <widget class="QLabel" name="MarkLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameTxt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MarkTxt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="OwnerLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Owner</string>
   </property>
  </widget>
  <widget class="QLabel" name="BirthDateLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BirthDate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PurposeTxt">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="InDateLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>InDate</string>
   </property>
  </widget>
  <widget class="QLabel" name="BirthPlaceLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BirthPlace</string>
   </property>
  </widget>
  <widget class="QLabel" name="BloodGroupLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BloodGroup</string>
   </property>
  </widget>
  <widget class="QLabel" name="FirstCalvingLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FirstCalving</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ColourTxt">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="InDateTxt">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>210</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="OwnerTxt">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ColourLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Colour</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BirthPlaceTxt">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="BirthDateTxt">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PurposeLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Purpose</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="FirstCalvingTxt">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="BloodGroupTxt">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
